src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public abstract double calculateArea();
+
+    public abstract void draw();

```

Comment:
these 2 methods will be inherited from the implemented interfaces and since this is an abstract class - you neither have to declare them nor to implement